engine/lib/phx/src/system/profiler.rs

```diff
@@ -16,8 +17,8 @@ pub type Signal = i32;
 pub struct Scope {
     pub name: *const libc::c_char,
     pub last: TimeStamp,
-    pub frame: u64,
-    pub total: u64,
+    pub frame: f64,
```

Comment:
do we really need a float here?

Reply:
Yes, it's a number of seconds with fractional milliseconds

Reply:
ah yeah it´s the delta time, for some reason thought this would just be a frame counter XD